### ISSUE TITLE
comment removed?

### ISSUE THREAD

**Issue by eworm-de** [Druvis-Timma/Mikrotik#2]
Hey Druvis,

thanks a lot for the video about Telegram chat. I left a lengthy comment with hints, concerns and a link to my own related project (https://github.com/eworm-de/routeros-scripts). It is no longer visible, so why was it removed?

**Comment by Druvis-Timma** [comment]
Just a policy to remove external links. Even if someone verified your link, the contents or link could be changed later and end up spreading malicious code. I admit I did not think it through when I asked if you have seen other better scripts.

**Comment by eworm-de** [comment]
The same happened to the comment to your second Telegram chat video. I did not give an url, but a hint how to find my project - probably that's too much already. I will not try again for your latest video.

Just a little side note... I have published my project in 2018 and improved it since then. Even [had a talk at MUM Europa 2019 in Vienna](https://www.youtube.com/watch?v=B9neG3oAhcY), though it evolved a lot since then. My Github project has over 700 stars, my Telegram group has nearly 500 members. Why should I add malicious code and ruin my reputation?

So you are still invited to have a look. Really, it's worth it:
https://github.com/eworm-de/routeros-scripts/blob/main/doc/telegram-chat.md
